rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java

```diff
@@ -65,9 +66,20 @@ public ClusterManagementServiceImpl(ConnectConfig connectConfig) {
      * @param connectConfig
      */
     private void prepare(ConnectConfig connectConfig) {
-        if (connectConfig.isAutoCreateGroupEnable()) {
-            ConnectUtil.createSubGroup(connectConfig, this.defaultMQPullConsumer.getConsumerGroup());
+        String consumerGroup = this.defaultMQPullConsumer.getConsumerGroup();
+        Set<String> topicSet = ConnectUtil.fetchAllTopicList(connectConfig);
```

Comment:
Is it possible to just check whether a single topic exists?

Reply:
> Is it possible to just check whether a single topic exists?

A ConnectController is created when the Runtime is started.When ConnectController created, the ClusterManagementServiceImpl, ConfigManagementServiceImpl, PositionManagementServiceImpl and OffsetMan AgementServiceImpl are created, in the process of creating the prepare method will check whether the corresponding topic exist if not then create it.

Reply:
What I mean is just to check if a single topic exists, no need to get the whole list of topics.